doc/rules/function_notation/static_lambda.rst

```diff
@@ -24,3 +24,5 @@ Example #1
    +$a = static function () use ($b)
     {   echo $b;
     };
+
+Tool `usage <./../../usage.rst>`_. All `built-in rules <./../../rules/index.rst>`_. All `rule sets <./../../ruleSets/index.rst>`_.
```

Comment:
what is this addition about ?

Reply:
it is a navigation item, I'm just trying something out to see if it can be improved

Reply:
reverted

Reply:
ah, i see. I am not against, just was trying to understand the 7k diff

Reply:
:+1:  we can always refine in a new PR, lets leave it out for now